### PR TITLE
Create actions for all components

### PR DESCRIPTION
**Pull request by Lenninlasd**
Separate the actions of the components, to improve readability and make the code more maintainable.

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -35,24 +38,31 @@ export const setNumberOfLights = value => ({
   value
 });
 
-export const setActividadRecinto = value => ({
-  type: SET_ACTIVIDAD_RECINTO,
+export const setRoomActivity = value => ({
+  type: SET_ACTIVIDAD_ROOM,
```
Comment (Newville23) [inline src/actions/index.js]: I think that this should be `SET_ACTIVITY_ROOM` , lmk what do you think about this @Lenninlasd .

File: src/actions/index.js [hunk 2]
```diff
@@ -35,24 +38,31 @@ export const setNumberOfLights = value => ({
...
   value
 });
 
-export const setTipoRecinto = value => ({
-  type: SET_TIPO_RECINTO,
+export const setRoomType = value => ({
+  type: SET_ROOM_TYPE,
   value
```
Comment (Newville23) [inline src/actions/index.js:48]: For actions this could be more verbose.

File: src/components/WallsConfig/index.js [hunk 3]
```diff
@@ -122,28 +123,22 @@ ConfigWalls.propTypes = {
 };
 
 const mapDispatchToProps = dispatch => ({
-  handleBackButton: () => dispatch({ type: 'HIDE_WINDOWS_PROPS' }),
+  handleBackButton: () => dispatch(hideMainFormLayout()),
```
Comment (Newville23) [inline src/components/WallsConfig/index.js:126]: Probably I'm wrong but I think would be better put this logic in the component as methods in order to just write something like 
```export default connect(
  mapStateToProps,
  {hideMainFormLayout, moreActions, etc}
)(MyComponent)
```

**Review by Newville23 — COMMENTED** [review]
Could you add a brief description to the PR ?

File: src/actions/index.js (hunk 1 above)
Comment (Lenninlasd) [reply src/actions/index.js]: Done.

File: src/actions/index.js (hunk 2 above)
Comment (Lenninlasd) [reply src/actions/index.js:48]: I don't find a more verbose one 😅.

File: src/components/WallsConfig/index.js (hunk 3 above)
Comment (Lenninlasd) [reply src/components/WallsConfig/index.js:126]: It could be done in future PRs.

**Pull request merged**